package.json

```diff
@@ -10,14 +10,17 @@
     "build-prod": "yarn run build:clean && NODE_ENV=production webpack -p --progress",
     "build-prod-readable": "yarn run build:clean && NODE_ENV=production webpack --progress",
     "build-docs": "documentation build src/types/profile.js -f md > docs/processed-profile-format.md; cat docs/processed-profile-format.md",
-    "lint": "eslint *.js src",
+    "lint": "eslint *.js bin src",
     "lint-fix": "yarn lint -- --fix",
     "flow": "flow",
     "flow-coverage": "flow-coverage-report -i 'src/**/*.js' -t html -t text",
     "flow-generate-libdefs": "flow-typed install",
     "license-check": "devtools-license-check",
+    "postcheckout": "node bin/post-checkout.js",
+    "postmerge": "node bin/post-merge.js",
     "precommit": "lint-staged",
     "prepush": "run-p flow license-check lint test",
+    "postrewrite": "node bin/post-rewrite.js",
     "publish": "rimraf public_html && cp -r dist public_html",
     "serve-static": "ws -d dist/ -s index.html -p 4242",
     "start": "mkdir -p dist && cp res/zee-worker.js dist/ && NODE_ENV=development node server.js",
```

Comment:
I tried to cover more cases than what's in the debugger. So here I get `postmerge` that's used when running `git pull`, `postcheckout` when moving between branches, and `postrewrite` when rebasing.

I still miss "reset" that I actually use a lot when updating a master but there is no git hook for this :/